### PR TITLE
PROGRAMMES-5864 Adding off-air badge to week schedule

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5864

Can compare nicely using http://www.bbc.co.uk/5livesportsextra/programmes/schedules/2013/w14 and http://sandbox.bbc.co.uk/schedules/p00fzl7h/2013-w14

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Organism/Broadcast/BroadcastPresenter.php [hunk 1]
```diff
@@ -19,43 +20,33 @@ class BroadcastPresenter extends Presenter
...
+    /** @var BroadcastInfoInterface */
     private $broadcast;
 
     /** @var CollapsedBroadcast */
     private $collapsedBroadcast;
 
     /**
      * BroadcastPresenter constructor.
-     * @param Broadcast|BroadcastGap $broadcast
+     * @param BroadcastInfoInterface $broadcast
```
Comment (BPScott) [inline src/Ds2013/Organism/Broadcast/BroadcastPresenter.php]: Now we can fully express the type contract in PHP do we need the docblock?

File: src/Ds2013/Page/Schedules/ByDayPage/SchedulesByDayPagePresenter.php [hunk 2]
```diff
@@ -194,11 +195,11 @@ public function hasSiblingServiceLink(): bool
      * Evening - 6pm until midnight
      * Late - midnight until 6am the next day
      *
-     * @param Broadcast|BroadcastGap $broadcast
+     * @param BroadcastInfoInterface $broadcast
```
Comment (BPScott) [inline src/Ds2013/Page/Schedules/ByDayPage/SchedulesByDayPagePresenter.php]: Now we can fully express the type contract in PHP do we need the docblock?

**Comment by votemike** [comment]
Test this please

**Pull request merged**